api/seeds/shared/roles-and-activities.js

```diff
@@ -32,8 +32,13 @@ const insertAndGetIDs = async (knex, tableName, values) => {
       return { name };
     });
 
-  await knex(tableName).insert(insert);
-  logger.info(`Completed seeding the ${chalk.cyan(tableName)} table`);
+  if (insert && insert.length > 0) {
```

Comment:
if (insert && insert.length){

is the same I think.  This is might be a bit more readable though.